### ISSUE TITLE
Main website map

### ISSUE THREAD

**Issue by elil** [pdxosgeo/foss4g2014-wordpress#64]
On the main website, the map has 6 (important) points.  To see the other map with more points, you need to click "view more".  Not sure if this is by design or not, if so, it might be good to suggest clicking the "view more" button gives additional information, not just a bigger screen.  Maybe the button should be "See more points of interest"

**Comment by twelch** [comment]
Yes it's by design.  It's meant to focus on the main conference locations.  Note, the Getting around Portland has a more detailed section now leading to this map (https://2014.foss4g.org/attending/getting-to-portland/).  I think if there's more explanatory text it should go here.
